Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1131,8 +1131,9 @@ def build_curl(self, text):
         """
         http_methods = ['GET', 'POST', 'PUT', 'DELETE', 'PATCH']
         data = text.split("send: b'")[1]
-        if data.startswith('{'):
+        if data[0] in {'{', '<'}:
```

Comment:
can data be empty? if so, then:
```suggestion
        if data and data[0] in {'{', '<'}:
```